src/scss/03-widgets/_popup.scss

```diff
@@ -50,3 +50,12 @@
 		}
 	}
 }
+
+// OS HighContrast Enabled -------------------------------------------------------
+///
+.os-hight-contrast {
+	.popup-dialog {
+		border: 3px solid #b7b7b7;
```

Comment:
```suggestion
		border: var(--border-size-l) solid #b7b7b7;
```

Reply:
Please, find the correct css variable for the color instead

Reply:
Done!